### ISSUE TITLE
dashboard-mode: Symbol’s function definition is void: linum-mode

### ISSUE THREAD

**Issue by k4r4b3y** [emacs-dashboard/emacs-dashboard#548]
Using emacs 29.4 installed from debian bookworm-backports.  I have also installed the `emacs-dashboard` from debian stable repo: https://packages.debian.org/bookworm/elpa-emacs-dashboard

I enable emacs-dashboard on my init.el file as follows:

```
(use-package dashboard
  :ensure nil
  :config
  (dashboard-setup-startup-hook))
```

And I start emacs.  However, in the *Messages* buffer I get the following error: "dashboard-mode: Symbol’s function definition is void: linum-mode"

The dashboard gets displayed in a broken manner similar to one reported here: https://github.com/emacs-dashboard/emacs-dashboard/issues/312#issue-878097112 --- I get ^L stuff and also my other emacs packages don't get loaded, it seems like the error interferes with the loading of my other emacs packages.

How do I resolve this?

**Comment by ricardoricho** [comment]
Hi, the debian repo is pointing to version 1.7 (3 years old). I suggest you install it from another source (manually, [melpa.org](https://melpa.org/#/), [straight](https://github.com/radian-software/straight.el), [jcs-elpa](https://jcs-emacs.github.io/jcs-elpa/#) or [elpaca](https://github.com/progfolio/elpaca)) to have the most recent version.

**Comment by k4r4b3y** [comment]
Is the problem related to the debian package being outdated?

**Comment by ricardoricho** [comment]
Yes, that version [disable lin-num](https://salsa.debian.org/emacsen-team/emacs-dashboard/-/blob/debian/master/dashboard.el?ref_type=heads#L58) (an external package) without checking if it's available.
